### DISCUSSION TITLE
Education draft icons

### DISCUSSION THREAD

**Discussion by connorliam** [lucide-icons/lucide#2443] [General]
I hope this is the right place for suggesting new icons. For a project I’m working on, I was searching for icons to represent “students,” “teachers,” and “classes,” but I couldn’t find any that fit. Where would be the best place to suggest these additions?

Additionally, if I were to create these icons myself, what guidelines should I follow, and how would I go about submitting them to be considered as part of the Lucide icon set?

Thank you!

![education draft icons](https://github.com/user-attachments/assets/e8738c18-9857-4d73-a9c6-aa0315aabb56)

**Comment by karsa-mistmere** [comment]
Hi @coding-wani 

See our [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide) and [contribution guide](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).

But these design ideas do seem a bit too dense to be feasible within these constraints.

**Reply by connorliam** [reply]
Thank you for the quick feedback and for sharing the design and contribution guidelines. 

I see now that my sketches are indeed too dense to fit within the constraints. I’m not sure how best to simplify the “students” and “classes” icons, but I have the impression that the “teacher” icon could potentially work by focusing on minimalism, perhaps just using the figure and the baston (and maybe a small part of board).

I hope some designer will be inspired by the challenge!
